docs/guides/migrations.md

```diff
@@ -10,28 +10,30 @@ kb_sync_latest_only
+If other header types are used in a project, these header types and the rendering need to be adapted accordingly.
 
 With the introduction of the new [navigation CMS components](../concepts/cms-integration.md#navigation-cms-components) it became necessary to adapt the main navigation styling.
-The styling can no longer rely on child selectors (`>`) since the CMS manged components introduce a lot of additional HTML tags around the styling relevant `li` tags.
+The styling can no longer rely on child selectors (`>`), since the CMS managed components introduce a lot of additional HTML tags around the styling-relevant `li` tags.
 
-To improve the performance of the 'Static Content Page' component the triggered `/pagetree` REST calls default behavior regarding the used `depth` was changed.
-In the past no given 'Navigation Depth' in the ICM backend configuration resulted in no limitation at all ("Define how many levels the navigation tree displays.
+To improve the performance of the 'Static Content Page' component, the default behavior of the triggered `/pagetree` REST call regarding the used `depth` has been changed.
+Previously, no 'Navigation Depth' specified in the Commerce Management configuration resulted in no limitation at all ("Define how many levels the navigation tree displays.
 To show all levels, leave the field empty.").
-With the content model adaptions of `icm-as-customization-headless:1.7.0` a depth default value of `5` was introduced and the description was changed accordingly.
-In the PWA the rendering was adapted as well so that an empty `NavigationDepth` value of the 'Static Content Page' component now defaults to `0` instead of no depth limitation, that resulted in the whole content page tree being fetched and saved to the state.
-To keep the current behavior in an existing project either adapt the `0` default in `pagelet.numberParam('NavigationDepth', 0)"` to a reasonable number or set the 'Navigation Depth' values for all 'Static Content Page' component instances in the ICM backend to reasonable depth values instead of leaving them empty.
+With the content model adaptions of `icm-as-customization-headless:1.7.0`, a depth default value of `5` has been introduced and the description has been changed accordingly.
+In the PWA, the rendering was adapted as well so that an empty `NavigationDepth` value of the 'Static Content Page' component now defaults to `0` instead of no depth limitation that resulted in the whole content page tree being fetched and saved to the state.
+To keep the current behavior in an existing project, either adapt the `0` default in `pagelet.numberParam('NavigationDepth', 0)"` to a reasonable number or set the 'Navigation Depth' values for all 'Static Content Page' component instances in Intershop Commerce Management to reasonable depth values instead of leaving them empty.
```

Comment:
Should 'Navigation Depth' and 'Static Content Page' also be formatted like `NavigationDepth` and `Static Content Page` or was it done differently here on purpose? (Also see line 33.)

Reply:
This was done intentionally. I use the code formatting if it is an actual value or name in the code, but when it is a display name in the ICM backoffice I used the single quotes. I could change it to double quotes if this helps anything?